### PR TITLE
Ollama ms docs and chatbot recipe udpates

### PR DESCRIPTION
**Pull request by Gregory-Pereira**
/cc @rhatdan @sallyom @lmilbaum 
ollama-ms changes:

- run model_server container in detached mode, path to default ollama config dir and add caveat about this in the docs

chatbot app makefile changes:

- rename variables for readability (ex: `MODELIMAGE` --> `MODEL_IMAGE`)
- reworked install for reliability and downloading from a trusted source
- fixing tests locally with python version collisions
- prepare variables for future image repo renaming/migration

### CODE REVIEW

**Review by lmilbaum — APPROVED** [review]
LGTM

File: recipes/natural_language_processing/chatbot/Makefile [hunk 1]
```diff
@@ -1,55 +1,94 @@
...
 
 .PHONY: functional_tests
 functional_tests:
-	python3 -m pytest -vvv --driver=Chrome --driver-path=./chromedriver tests/functional
+	pytest -vvv --driver=Chrome --driver-path=$(CHROMEDRIVER_EXISTS) tests/functional
 
 .PHONY: integration_test
 integration_tests:
-	URL=${URL} python3 -m pytest -vvv --driver=Chrome --driver-path=./chromedriver tests/integration
+	URL=${URL} pytest -vvv --driver=Chrome --driver-path=$(CHROMEDRIVER_EXISTS) tests/integration
```
Comment (rhatdan) [inline recipes/natural_language_processing/chatbot/Makefile]: This will blow up it CHROMEDRIVER does not exists?

Comment (Gregory-Pereira) [reply recipes/natural_language_processing/chatbot/Makefile]: Yes it does, good catch, i'll fix this.

**Comment by Gregory-Pereira** [comment]
I have rebased and made some updates to keep in alignment with the new `common` recipes directory and makefile:

- Variables renamed from `ONEWORDVARS` to `ONE_WORD_VARS` for readability
    - If we are strongly against this thats ok but reviewing these PRs is killing dyslexia 
- Removing chatbot specific make targets from common, ex:
    - `make install` which installs the chromedriver only used in the tests for the `chatbot` app should not be in common
- Common additions:
    - run should use the `PORT` env variable because each application may have its own default port
    - clean target should have a step to remove `pytest` files that wont fail if they dont exist 
- workflow / testing change: changed chatbot testing make targets to be same style as rest of repo, ex: `test_name` --> `test-name` so its the same as the `model_servers` tests

File: recipes/natural_language_processing/chatbot/Makefile [hunk 2]
```diff
@@ -1,3 +1,45 @@
 APP ?= chatbot
+PORT ?= 8501
 
 include ../../common/Makefile.common
+
+CHROMEDRIVER_EXISTS ?= $(shell command -v chromedriver)
```
Comment (rhatdan) [inline recipes/natural_language_processing/chatbot/Makefile]: Should this all be in Common?

Comment (Gregory-Pereira) [reply recipes/natural_language_processing/chatbot/Makefile]: If there are other recipes that use `chromedriver` then most definitely. If there are not it is considered specific to  this recipe. I couldn't find any other references to `chromedriver` so I stuck this here.

Comment (Gregory-Pereira) [reply recipes/natural_language_processing/chatbot/Makefile]: As per your and Michaels comments I have abstracted the chromedriver instllation into common. Now only each recipe will provide the relative installation path to the recipes dir for installation.

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**